content/documentation/pages/5-feature-guides/3-batch/7-java-dsl.md

```diff
@@ -168,7 +170,9 @@ Map<String, String> taskLaunchProperties = new DeploymentPropertiesBuilder()
 long executionId = task.launch(taskLaunchProperties, Collections.EMPTY_LIST);
 ```
 
-### Setting Properties for a TaskSchedule
+### Setting Properties for `TaskSchedule`
+
+Similarly, you can set deployment properties for a `TaskSchedule` instance:
```

Comment:
I believe we now have a consolidated approach to passing deployer properties for scheduling. @cppwfs: Could you please review this for correctness?